lib/index.js

```diff
@@ -78,8 +73,12 @@ export default {
         const fileText = TextEditor.buffer.cachedText;
 
         // Is flow enabled for current file ?
-        if (fileText.indexOf('@flow') === -1 && !this.enableAll) {
-          return [];
+        const firstComStart = fileText.indexOf('\/*');
```

Comment:
What was wrong with the older check?

You could maybe replace the test text with `/** @flow */`
I haven't seen any other so far